Lesson_24_Loops/js/homework.js

```diff
@@ -0,0 +1,150 @@
+//1. Подсчитать сумму всех чисел в заданном пользователем диапазоне.
+const num1 = +prompt("Начало диапазона:");
+const num2 = +prompt("Конец диапазона:");
+
+let sum = 0;
+let i = num1;
+
+while (num1 <= num2) {
+  sum += i;
+  i++;
+}
+alert(sum);
```

Comment:
Здесь получится бесконечный цикл, потому что ты увеличиваешь `i++;`, но в условии у тебя остаётся `num1 <= num2`, которые никак не меняются.
```suggestion
const num1 = +prompt("Начало диапазона:");
const num2 = +prompt("Конец диапазона:");

let sum = 0;
let i = num1;

while (i <= num2) { 
  sum += i;
  i++;
}
alert(sum);
```